src/frontend/web_application/src/components/MessageList/components/Message/style.scss

```diff
@@ -1,107 +1,102 @@
 @import '../../../../styles/vendor/bootstrap_foundation-sites';
 @import '../../../../styles/object/o-avatar';
+@import '../../../../styles/object/o-reset-html-message';
 
 .m-message {
   @include flex-grid-row($size: expand);
+  margin-bottom: $co-grid__gutter--small;
   font-size: $co-font__size--small;
 
-  &__content { @include flex-grid-size; }
+  &__avatar-col {
+    @include flex-grid-column(shrink);
+    display: none;
+    padding-top: $co-component__spacing;
+  }
+
+  &__avatar { @include m-avatar--size(medium); }
+
+  &__container {
+    @include flex-grid-column;
+    padding-right: 0;
+    padding-left: 0;
+    background: $co-color__fg__back;
+  }
 
   &__top-bar {
     @include flex-grid-row($size: expand);
-    justify-content: center;
+    align-items: center;
     border-bottom: 1px solid $co-color__fg__back--lower;
     background: $co-color__fg__back--higher;
   }
 
+  &__type {  @include flex-grid-size(shrink); }
+  &__type-icon { color: $co-color__fg__text; }
+  &__type-label { display: none; }
+
+  &__pi { @include flex-grid-size(shrink); }
+
+  &__author {
+    @include flex-grid-column;
+    color: $co-color__fg__text--high;
+    font-weight: 600;
+  }
+
+  &__date {
+    @include flex-grid-column(shrink);
+    margin-left: auto;
+  }
+
   &__actions-switcher {
     @include flex-grid-size(shrink);
     border-left: 1px solid $co-color__fg__back--lower;
   }
 
   &__actions-menu { background: $co-color__fg__back--higher; }
 
+
+  &__subject {
+    padding: $co-component__spacing;
+    border-bottom: 1px solid $co-color__fg__back--lower;
+    color: $co-color__fg__text--lower;
+  }
+
   &__body {
-    @include flex-grid-column(12);
-    padding-top: $co-component__spacing;
-    padding-bottom: $co-component__spacing;
-    background: $co-color__fg__back;
+    padding: $co-component__spacing;
     overflow: hidden;
+    color: $co-color__fg__text;
 
-    &__content {
-      padding: $co-component__spacing 0;
-      color: $co-color__fg__text--low;
+    &--exerpt {
+      &,
+      > * {
+        display: inline-block;
+      }
 
       &::after {
```

Comment:
excerpt should be a 200chars max length. 
the height of the excerpt div should be 1 line, so we can hide the rest and show an ellipsis when text is longer